src/ripple/app/consensus/RCLValidations.h

```diff
@@ -152,6 +152,7 @@ class RCLValidatedLedger
     using Seq = LedgerIndex;
     struct MakeGenesis
     {
+        explicit MakeGenesis() = default;
```

Comment:
I'd probably leave tag types alone.

Reply:
Tag types need it most.  It has become std::practice to do this.  See for example `allocator_arg_t`.  The problem is this:
```c++
void some_func(other_type = default_value);
void some_func(tag_type, other_type = default_value);
// ...
some_func({});
```
`{}` can become ambiguous as to whether it applies to `other_type` or `tag_type`.  If you're lucky it is a compile-time error.

https://wg21.link/lwg2510 has a better description of the issue.

Reply:
So the most important part of this change is making the various default constructors `explicit.  Did I get that right?

Reply:
Right.  That's about the most helpful way to turn off "aggregateness".  That's not to say that all default constructors should be implicit though, though I would say that is a good rule of thumb for tag types.